### PR TITLE
DOCS-5584 brand intro

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Based on review in https://github.com/mulesoft/docs-api-community-manager/pull/51

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
@kbostrom , please only review the edits to these files, and not the unedited parts of the files which will be covered later.

File: modules/ROOT/pages/brand-intro.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
 endif::[]
```
Comment (kbostrom) [inline modules/ROOT/pages/brand-intro.adoc:4]: NOTE:
Very quick copyedit of changed text only.

File: modules/ROOT/pages/configure-acm.adoc [hunk 2]
```diff
@@ -279,7 +279,7 @@ If you want to organize CMS content using Topics:
 
 === Enable Chatter Tracking for APIs [[enable-chatter-tracking]]
 
-Chatter tracking allows creating discussions on specific objects like APIs. To use Chatter tracking for APIs:
+Chatter tracking enables creating discussions on specific objects like APIs. To use Chatter tracking for APIs:
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
Chatter tracking enables your community to create discussions on specific objects like APIs. To use Chatter tracking for APIs:
```
enable people

File: modules/ROOT/pages/configure-acm.adoc [hunk 3]
```diff
@@ -433,7 +433,7 @@ image::access-mapping.png[Access Mapping for CommunityApi]
 
 ==== Add Self Register Component to Registration Page [[add-self-register-custom-component-to-register-page]]
 
-The self-register component allows guest users to self-register in your community to become members. If you're not using any of the ACM out-of-the-box templates, follow these steps to use the ACM-specific self-registration component.
+The self-register component enables guest users to self-register in your community to become members. If you're not using any of the ACM out-of-the-box templates, follow these steps to use the ACM-specific self-registration component.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
The self-register component enables guest users to register themselves as members of your community. If you're not using any of the ACM out-of-the-box templates, follow these steps to use the ACM-specific self-registration component:
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -15,7 +15,7 @@ See these useful resources to get up-to-speed with the capabilities of Salesforc
 
 == What is an API Community?
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
== What Is an API Community?
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -15,7 +15,7 @@ See these useful resources to get up-to-speed with the capabilities of Salesforc
...
 
-To drive developer adoption of your APIs, you must help them discover and learn how to use your APIs effectively, and provide a personalized, branded experience to motivate them to work with you. You must also engage with these developers during every step of their journey to ensure their success. ACM enables you to create this personalized, connected experience for your developers to allow them to interact with your APIs - APIs designed, catalogued, and managed in the Anypoint Platform.
+To drive developer adoption of your APIs, you must help them discover and learn how to use your APIs effectively, and provide a personalized, branded experience to motivate them to work with you. You must also engage with these developers during every step of their journey to ensure their success. ACM enables you to create this personalized, connected experience for your developers so they can interact with your APIs - APIs designed, catalogued, and managed in the Anypoint Platform.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
To drive developer adoption of your APIs, you must help them discover and learn how to use your APIs effectively, and provide a personalized, branded experience to motivate them to work with you. You must also engage with these developers during every step of their journey to ensure their success. ACM enables you to create this personalized, connected experience for your developers so they can interact with your APIs: APIs designed, cataloged, and managed in Anypoint Platform.
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -33,7 +33,7 @@ Other ACM features enable your organization to engage with partners and ensure t
 
 == Building Your API Community
 
-Personalize your community look and feel, branding, and site structure with the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank], a drag-and-drop editor that allows you to create and customize the experience down to the finest level of detail. The Community Builder makes it easy to create a tailored experience using pre-existing components and simple clicks, or using the full flexibility of HTML and CSS for precise pixel-perfect experiences.
+Personalize your community look and feel, branding, and site structure with the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank], a drag-and-drop editor that enables you to create and customize the experience at every level of detail. The Community Builder makes it easy to create a tailored experience using pre-existing components and simple clicks, or using the full flexibility of HTML and CSS for precise pixel-perfect experiences.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Personalize your community look and feel, branding, and site structure by using  https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank], a drag-and-drop editor that enables you to create and customize the user experience at every level of detail. Community Builder makes it easy to create a tailored experience using pre-existing components and simple clicks, or using the full flexibility of HTML and CSS for precise pixel-perfect experiences.
```
This is sounding a bit Markety...careful!  :-)

**Review by kbostrom — APPROVED** [review]
Here's the third!

**Pull request merged**